liquibase-core/src/main/java/liquibase/command/core/GenerateChangelogCommandStep.java

```diff
@@ -1,78 +1,166 @@
-                .setValueObfuscator(ConfigurationValueObfuscator.STANDARD)
-                .build();
-        DRIVER_ARG = builder.argument("driver", String.class)
-                .description("The JDBC driver class").build();
-        DRIVER_PROPERTIES_FILE_ARG = builder.argument("driverPropertiesFile", String.class)
-                .description("The JDBC driver properties file").build();
-        CHANGELOG_FILE_ARG = builder.argument(CommonArgumentNames.CHANGELOG_FILE, String.class).required()
-                .description("File to write changelog to").build();
-        DATA_OUTPUT_DIRECTORY = builder.argument("dataOutputDirectory", String.class)
+        final CommandBuilder builder = new CommandBuilder(COMMAND_NAME);
+        CHANGELOG_FILE_ARG = builder.argument(CommonArgumentNames.CHANGELOG_FILE, String.class)
+                .description("Changelog file to write results").build();
+
+        AUTHOR_ARG = builder.argument("author", String.class).hidden().build();
+        CONTEXT_ARG = builder.argument("context", String.class).hidden().build();
+        DATA_OUTPUT_DIR_ARG = builder.argument("dataOutputDirectory", String.class)
                 .description("Directory to write table data to").build();
-        EXCLUDE_OBJECTS_ARG = builder.argument("excludeObjects", String.class)
-                .description("Objects to exclude from diff").build();
-        INCLUDE_OBJECTS_ARG = builder.argument("includeObjects", String.class)
-                .description("Objects to include in diff").build();
-        INCLUDE_TABLESPACE_ARG = builder.argument("includeTablespace", String.class)
-                .description("Include the tablespace attribute in the changelog").build();
-        SCHEMAS_ARG = builder.argument("schemas", String.class)
-                .description("Schemas to include in diff").build();
-        INCLUDE_SCHEMA_ARG = builder.argument("includeSchema", Boolean.class)
-                .defaultValue(false)
-                .description("If true, the schema will be included in generated changeSets").build();
-        INCLUDE_CATALOG_ARG = builder.argument("includeCatalog", Boolean.class)
-                .defaultValue(false)
-                .description("If true, the catalog will be included in generated changeSets").build();
-        DIFF_TYPES_ARG = builder.argument("diffTypes", String.class)
-                .description("Types of objects to compare").build();
-        OVERWRITE_OUTPUT_FILE_ARG = builder.argument("overwriteOutputFile", String.class)
-                .description("Flag to allow overwriting of output changelog file").build();
+        OVERWRITE_OUTPUT_FILE_ARG = builder.argument("overwriteOutputFile", Boolean.class)
+                .defaultValue(false).description("Flag to allow overwriting of output changelog file. Default: false").build();
+
+        // hiding parameter names that are not available externally but are used by this step.
```

Comment:
The first line of this comment should probably be at line 53, prior to the first hidden parameter. The second sentence can stay here, as it's useful information.